packages/warehouses/src/warehouseClients/SnowflakeWarehouseClient.ts

```diff
@@ -462,9 +462,9 @@ export class SnowflakeWarehouseClient extends WarehouseBaseClient<CreateSnowflak
         const whereSql = databaseName ? `AND TABLE_CATALOG ILIKE ?` : '';
         const query = `
             SELECT
-                LOWER(TABLE_CATALOG) as "table_catalog",
-                LOWER(TABLE_SCHEMA) as "table_schema",
-                LOWER(TABLE_NAME) as "table_name"
+                TABLE_CATALOG as "table_catalog",
```

Comment:
This was the root cause of the bug - in the rest of Lightdash we force lowercase identifiers. But for the SQL Runner this is causing errors. Now all case is handled as case sensitive (see screenshots in PR)